### PR TITLE
proxies the remoteMethod function

### PR DESCRIPTION
**Pull request by BoLaMN**
proxies the remoteMethod function to catch any remoteMethods that are added after this mixin has been attached.

### CODE REVIEW

File: remote-routing.js [hunk 1]
```diff
@@ -8,6 +8,26 @@ module.exports = function(Model, options) {
...
 };
 
+var remoteMethod = Model.remoteMethod;
+
+Model.remoteMethod = function(name, config) {
+  var disable;
+
+  remoteMethod.call(Model, name, config);
+
+  if (options.only && options.only.length) {
```
Comment (Neil-UWA) [inline remote-routing.js]: I suppose options should be config, right?

**Comment by Neil-UWA** [comment]
nice patch, I'm looking into it.

File: remote-routing.js (hunk 1 above)
Comment (BoLaMN) [reply remote-routing.js]: options is the mixin options that gets passed though to check against, config is the remoteMethod config 

eg. 

Model.remoteMethod('setup', { 'accepts': [] })

**Comment by BoLaMN** [comment]
sorry indent got me! fixed now

**Comment by Neil-UWA** [comment]
@BoLaMN  LTGM. merged.  I will release it later once I tested it thoroughly. Thanks for your efforts.

**Comment by Neil-UWA** [comment]
@BoLaMN There are some issues with the pr, It takes me sometime to solve it before releasing it. I only have time to work on this at night, sorry for the delayed release.

**Comment by Neil-UWA** [comment]
@BoLaMN  Released, you can checkout the change.md and readme

**Pull request merged**